### ISSUE TITLE
Update to latest version of the driver

### ISSUE THREAD

**Issue by kpaxton** [abachar/chrome-driver-standalone#2]
Can you update this to include the latest version of the driver?
I'm getting issues with 2.28.

```
WebDriverError: unknown error: unrecognized Chrome version: HeadlessChrome/59.0.3071.115
```

Since I am running Chrome 59. I looks like I would need 2.31 to be specific. 

If there is any way for you to update your versions to work with all of the later versions that would be great. 
2.29
2.30
2.31
2.32
2.33
2.34

**Comment by abachar** [comment]
Yes of course, will be done this week end

**Comment by abachar** [comment]
Sorry for the delay, santa arrived late :)
I pushed versions from 2.29 to 2.33, for v2.34 google didn't release linux32 version, i need more time

**Comment by kpaxton** [comment]
Awesome! thank you!

**Comment by Jagget** [comment]
https://github.com/abachar/chrome-driver-standalone/pull/3

**Comment by Jagget** [comment]
https://github.com/abachar/chrome-driver-standalone/pull/4

**Comment by Jagget** [comment]
https://github.com/abachar/chrome-driver-standalone/pull/5

**Comment by abachar** [comment]
fixed by @Jagget in #4
